app/views/events/show.html.erb

```diff
@@ -16,9 +16,10 @@
       </div>
       <div class="panel-body">
         <% if @event.owner %>
-          <%= link_to(url_for_twitter(@event.owner)) do %>
-            <%= image_tag @event.owner.image_url %>
-            <%= "@#{@event.owner.nickname}" %>
+          <% if @event.owner.provider == 'twitter' %>
+            <%= link_to "@#{@event.owner.nickname}", url_for_twitter(@event.owner) %>
```

Comment:
__[brakeman]__
* Unsafe model attribute in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)